test/state/state.hpp

```diff
@@ -66,6 +66,8 @@ class State
     [[nodiscard]] const auto& get_accounts() const noexcept { return m_accounts; }
 };
 
+using Withdrawals = std::vector<std::pair<address, uint64_t>>;
```

Comment:
Not necessary but perhaps a comment explaining what the value is and what range it has is useful.

Reply:
done

Reply:
I think better will be to just define single `using Withdrawal = std::pair<address, uint64_t>`. But at this point a struct with named fields will do better.